cpu/stm32/periph/timer.c

```diff
@@ -22,7 +22,6 @@
 
 #include "cpu.h"
 #include "periph/timer.h"
-#include <sys/_intsup.h>
```

Comment:
This looks like `clangd` header insertion artifact. I had a lot of those until I finally just globally disabled it. Maybe we should add some sane default clangd configuration to the repo, so that others don't run into the same issue.